### PR TITLE
Added print documentation

### PR DESCRIPTION
**Pull request by ZsoltBalvanyos**

### CODE REVIEW

File: src/main/tut/docs/channels.md [hunk 1]
```diff
@@ -9,27 +9,29 @@ Comms can be issued over the following channels:
...
 
-A template obviously must have at least one channel, which can be either SMS or email. 
+A template obviously must have at least one channel, which can be either SMS, email or print. 
 
 ## Customer Communication Preferences
 
 Within My OVO customers set which channels they wish to receive comms over, they are able to do this for Service and Marketing comms separately.
  
-For Service comms the customer is forced to choose either Post or Email, with SMS available as an additional option.
+For Service comms the customer is forced to choose either Print or Email, with SMS available as an additional option.
```
Comment (filosganga) [inline src/main/tut/docs/channels.md]: This conflict with other statements.

File: src/main/tut/docs/channels.md [hunk 2]
```diff
@@ -9,27 +9,29 @@ Comms can be issued over the following channels:
...
 
 Unless specified by the customer in their communication preferences, comms will not be issued via said channel.
 
 <div class="alert alert-info">
 If a customer has no Service Communication Preferences this indicates that they have not actually set any, as either email or post must be selected, and therefore the customer's Communication Preferences are not taken into account.
 </div>
 
+[comment]: <> (This statement has to be reviewed.)
 While the platform is only supporting notification comms, and only supporting email and SMS channels, if a customer has set their Service Communication Preferences to post rather than email, then post is removed from their list of Service Communication Preferences used by the platform.
```
Comment (laurence-bird) [inline src/main/tut/docs/channels.md]: This is now only applicable for comms being sent to a customer (with a customerId)

File: src/main/tut/docs/templates.md [hunk 3]
```diff
@@ -75,6 +75,75 @@ The sender of the SMS is currently set by the platform.
...
+    - county
+* Optional
+    - line 2
+    - country
+			
+For print
+ (`<div id="letterAddress"> `)
+##### Footers
+
+To add footer for every page of a letter, the content of the footer has to be encapsulated by the <footer> tag. It is importent that the <footer> element has to be the first element in the body.
```
Comment (laurence-bird) [inline src/main/tut/docs/templates.md]: typo here- *important

File: src/main/tut/docs/templates.md [hunk 4]
```diff
@@ -75,6 +75,75 @@ The sender of the SMS is currently set by the platform.
...
+    border-radius: 10px;
+    border: solid cmyk(30%, 0%, 40%, 0%) 2px;
+    padding: 10px;
+    padding-top: 0px;
+}
+```
+
+##### Assets
+Similarly to email, a print template can also reference optional assets, however images have to meet the following criterias:  
+ - encoded in a CMYK colourspace supporting format, such as tiff or jpg.
```
Comment (laurence-bird) [inline src/main/tut/docs/templates.md]: I think we could be more explicit here about the image format we support as this sounds like we may allow other file formats. Something like:

- either jpg or tiff images,  encoded in a CMYK colour space

File: src/main/tut/docs/templates.md [hunk 5]
```diff
@@ -75,6 +75,75 @@ The sender of the SMS is currently set by the platform.
...
+    padding-top: 0px;
+}
+```
+
+##### Assets
+Similarly to email, a print template can also reference optional assets, however images have to meet the following criterias:  
+ - encoded in a CMYK colourspace supporting format, such as tiff or jpg. 
+ - its size is set to ensure that it fits onto a page, if an image is wider or higher than the page, the template will be rejected.
+
+Templates cannot contain scripts and cannot reference third party stylesheets.
```
Comment (laurence-bird) [inline src/main/tut/docs/templates.md]: perhaps we should specify here that if they want to include stylesheets, they should be provided as assets

**Pull request merged**